### PR TITLE
Added Doc.md for game_of_fifteen bot

### PR DESCRIPTION
**Pull request by anehls93**

### CODE REVIEW

File: zulip_bots/zulip_bots/bots/game_of_fifteen/doc.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# game_of_fifteen bot
```
Comment (eeshangarg) [inline zulip_bots/zulip_bots/bots/game_of_fifteen/doc.md]: Replace this with `Game of Fifteen Bot` :)

File: zulip_bots/zulip_bots/bots/game_of_fifteen/doc.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Game of Fifteen bot is a bot designed to let you play a game of fifteen. To summon it, simply type `@game_of_fifteen`,
+like so:
+
```
Comment (eeshangarg) [inline zulip_bots/zulip_bots/bots/game_of_fifteen/doc.md]: Make this:

```
This bot is designed to let you play a [game of fifteen](https://en.wikipedia.org/wiki/15_puzzle).
To start a new game, simply type:
```

File: zulip_bots/zulip_bots/bots/game_of_fifteen/doc.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+```
+@game_of_fifteen
+```
+
+Run this bot as described in [here](https://zulipchat.com/api/running-bots#running-a-bot).
```
Comment (eeshangarg) [inline zulip_bots/zulip_bots/bots/game_of_fifteen/doc.md]: Replace `as described in` with just `as described`. :)

File: zulip_bots/zulip_bots/bots/game_of_fifteen/doc.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+## Usage
+
+The goal of the game is to arrange all numbers from smallest to largest, starting with the grey question mark in the upper left corner, and then moving through each row until we hit the end.
+
```
Comment (eeshangarg) [inline zulip_bots/zulip_bots/bots/game_of_fifteen/doc.md]: Move this paragraph to the top of the previous section. Also, replace `we hit the end` with `till one reaches the end`.

**Comment by PIG208** [comment]
We might want to close this PR as the owner of it is no longer active.

**Comment by eeshangarg** [comment]
@PIG208: Yeah, we can close this one.

@anehls93 Since this has been inactive for a while, I am going to go ahead and close this. Thanks so much for contributing to Zulip, we appreciate it! :)

**Pull request closed without merging**